### ISSUE TITLE
Second round modulo wrong

### ISSUE THREAD

**Issue by HenerHoop** [renekorss/personal-id-code-php#5]
Example ID code: 38605230034

getControlNumber returns wrong value.

**Fix (maybe):**

Reset $total before secondary loop
```php
// Second round
if ($modulo === 10) {
    $total = 0;

    for ($i = 0; $i < 10; $i++) {
        $multiplier = $i + 3;
        $total += substr($code, $i, 1) * ($multiplier > 9 ? $multiplier - 9 : $multiplier);
    }

    $modulo = $total % 11;
     if ($modulo === 10) {
         $modulo = 0;
     }
}
```

**Comment by renekorss** [comment]
Thanks for reporting. Will take a look soon.

**Comment by renekorss** [comment]
@HenerHoop Fixed in [1.1.2](https://github.com/renekorss/personal-id-code-php/releases/tag/1.1.2). Thanks again!
